Formula/ani-cli.rb

```diff
@@ -0,0 +1,18 @@
+class AniCli < Formula
+  desc "A cli tool to browse and play anime"
+  homepage "https://github.com/pystardust/ani-cli/"
+  sha256 "b53a8ebf7bf44bc7497ad4b4247e941ca9a7a89365717d3f4a2da349bcc80e45"
+  url "https://github.com/pystardust/ani-cli/archive/refs/tags/v1.6.tar.gz"
+  license "GPL-3.0"
+
+  depends_on "grep"
+  depends_on "curl"
+  depends_on "openssl"
+  depends_on "cask"
+  depends_on "aria2"
+  depends_on "mpv"
+  
+  def install
```

Comment:
It needs to have an option to disable auto-update. https://docs.brew.sh/Acceptable-Formulae#we-dont-like-tools-that-upgrade-themselves

Reply:
We would prefer not to maintain seperate branches of our project, as that has proven difficult in the past.
Is it possible to provide a patch file disableing `-U` in our script?